frontend/src/components/Review/ReviewList.vue

```diff
@@ -0,0 +1,210 @@
+            <div class="review-title">
+                <div class="left-side-review-title">
+                    <img src="https://ss1.4sqi.net/img/venuepage/v2/section_title_tips@2x-6449ea09a26b1d885184e709e2c8f693.png" height="25" width="25">
+                    <span>21 Reviews</span>
+                </div>
+                <div class="review-title-search">
+                    <div class="control has-icons-left">
+                        <input class="input is-small" type="search" placeholder="Find review..">
+                        <span class="icon is-small is-left">
+                            <i class="fas fa-search"></i>
+                        </span>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div class="add-review-wrp">
+            <div class="add-review">
+                <div class="left-side-add-review">
+                    <img src="https://ss0.4sqi.net/img/venuepage/v2/add_tip_blank_avatar@2x-4321684c656168f26ae9208901a9d83e.png" height="32" width="32">
+                    <span><a>Login</a>, to leave a review.</span>
+                </div>
+                <div class="add-review-btn">
+                    <button class="button" disabled>Post</button>
+                </div>
+            </div>
+        </div>
+        <div class="reviews-section-wrp">
+            <div class="reviews-section-header">
+                <div class="filter-area">
+                    <ul>
+                        <li class="sort-word">Sort by:</li>
+                        <li v-on:click="onSortFilter('popular')" :class="{ active: isActive.popular }"><a>Popular</a></li>
+                        <li v-on:click="onSortFilter('recent')" :class="{ active: isActive.recent }"><a>Recent</a></li>
+                    </ul>
+                </div>
+            </div>
+            <div class="reviews-section-list">
+                <template>
+                    <Review></Review>
+                    <Review></Review>
+                    <Review></Review>
+                </template>
+            </div>
+        </div>
+    </div>
+</template>
+
+<script>
+    import Review from './ReviewListElement';
+
+    export default {
+        components: {
+            Review
+        },
+
+        data() {
+            return {
+                isActive: {
+                    popular: true,
+                    recent: false
+                }
+            };
+        },
+
+        computed: {
+        },
+
+        methods: {
+            onSortFilter(name) {
+                for (let item in this.isActive) {
+                    if (item === name) {
+                        this.isActive[item] = true;
+                    } else {
+                        this.isActive[item] = false;
+                    }
+                }
+            }
+        }
+    }
+
+</script>
+
+<style scoped>
+
+    /* Main block wrapper. */
+    .reviews-wrp {
+        width: 40%;
```

Comment:
All looks great except width: 40%, I think parent component has to set width of child

Reply:
I added it for nice viewing during creating/checking.
I think so too and remove that.